templates/web/base/waste/bulky/items.html

```diff
@@ -9,15 +9,33 @@
 [% IF property %]
   [% INCLUDE 'waste/_address_display.html' %]
 [% END %]
-
+[% IF cobrand.moniker == 'sutton' || cobrand.moniker == 'kingston' %]
+[% SET cfg = c.cobrand.wasteworks_config %]
+<div
+  data-base-price="[% cfg.base_price %]"
+  data-band1-price="[% cfg.band1_price %]"
+  data-band1-max="[% cfg.band1_max %]"
+  data-items-per-collection-max="[% cfg.items_per_collection_max %]"
+  id="band_pricing_data">
+</div>
```

Comment:
This data is already included (though in a slightly different format) in the `$('#js-bulky-total').data('pricing')` so it might make sense to use the same source?